sonar-python-plugin/src/test/java/org/sonar/plugins/python/PythonSquidSensorTest.java

```diff
@@ -73,15 +73,16 @@ public void should_execute_on_python_project() {
     Project project = mock(Project.class);
     assertThat(sensor.toString()).isEqualTo("PythonSquidSensor");
     assertThat(sensor.shouldExecuteOnProject(project)).isFalse();
-    fs.add(new DefaultInputFile("test.py").setLanguage(Python.KEY));
+    fs.add(new DefaultInputFile(TestUtils.MODULE, "test.py").setLanguage(Python.KEY));
```

Comment:
I did not know what to do with moduleKey, so I centralized its value in one place.